frontend/packages/ux-editor/src/components/Elements/Subform/CreateSubformWrapper.tsx

```diff
@@ -52,14 +61,17 @@ export const CreateSubformWrapper = ({
           value={newSubformName}
           onChange={(e) => onNameChange(e.target.value)}
           error={nameError}
+          disabled={isPendingLayoutSetMutation}
         />
         <StudioButton
           className={classes.confirmCreateButton}
           variant='secondary'
-          onClick={onCreateConfirmClick}
+          onClick={() =>
```

Comment:
Should we refactor this to a `handleClick` function?

Reply:
I understand the suggestion, but I don't see the need to refactor it into a separate function since it's just a one-liner. However, I'm open to making the change if you think it's necessary 🙂.

Reply:
See our discussion on Slack😊